egs2/fsc/asr1/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="valid"
+test_sets="test valid"
+
+asr_config=conf/tuning/train_asr_conformer7_n_fft512_hop_length256.yaml
```

Comment:
I would recommend you to use a soft link at `conf` as `train_asr.yaml` to make it standardized. (same for lm and decoding)